libs/design/src/atoms/form/form-field/form-field/form-field.component.html

```diff
@@ -1,2 +1,3 @@
-<ng-content select="[daff-input]"></ng-content>
+<ng-content select="input"></ng-content>
+<ng-content select="select"></ng-content>
```

Comment:
I think this should only transclude `[daff-input]` and `[daff-select]`

Reply:
fixed in 996ec6b